config/common.py

```diff
@@ -57,11 +57,14 @@
                 },
                 {
                     'type': 'httpjson',
-                    'url': 'http://httpjson-server:12345/rfm',
+                    'url': 'http://log.cscs.ch:31311/reframe',
```

Comment:
Leave this unchanged.

Reply:
Where is `http://httpjson-server:12345/rfm` being used ?
`common.py` is for CSCS so I would rather keep it.